bin/crew

```diff
@@ -1817,13 +1817,10 @@ def upload_command(args)
 end
 
 def upstream_command(args)
-  puts "#{'Package'.ljust(35)}#{'Status'.ljust(20)}#{'Current'.ljust(20)}Upstream"
-  puts "#{'-------'.ljust(35)}#{'------'.ljust(20)}#{'-------'.ljust(20)}--------"
   args['<name>'].each do |name|
-    name.gsub!('.rb', '')
-    search name
+    search name unless name.include?('*')
```

Comment:
No!  I have told you repeatedly the `args['<name>']` output is an array of package files.  The original arg with the `*` has been replaced.

Reply:
I have tested this code and it works. If you encounter an error locally, please tell me.

Reply:
CREW_REPO=https://github.com/Zopolis4/chromebrew.git CREW_BRANCH=hath crew update && crew upstream z*

Reply:
Do me a favor.  Add `abort args['<name>'].to_s` and notice the output.  There is no `*` argument!